### DISCUSSION TITLE
sidecar-browsershot

### DISCUSSION THREAD

**Discussion by stefanzweifel** [HassanZahirnia/laravel-package-ocean#68] [Package Suggestions]
A package that allows one to run spatie/browsershot on AWS Lambda through Sidecar.

- https://github.com/stefanzweifel/sidecar-browsershot
- https://packagist.org/packages/wnx/sidecar-browsershot

**Comment by HassanZahirnia** [comment]
Thanks for the suggestion 😊 I've added the package! It will be live in a moment.
